.github/workflows/cd.yml

```
@@ -167,15 +167,16 @@ jobs:
       matrix:
         core: ['', 'true']
         perf: ['', 'true']
+        python-env: [3.7, 3.8, 3.9, 3.10, 3.11]
```

```suggestion
        python-env: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

this is needed otherwise it try to install python 3.1 instead of 3.10